### DISCUSSION TITLE
Switch display OFF vs brightness to 0

### DISCUSSION THREAD

**Discussion by rebel7580** [HASwitchPlate/HASPone#136] [Q&A] [answered]
Under certain conditions (i.e., no one in the room to see it), I want the display "off". I can do this either by sending a  .../light/switch OFF or a .../brightness/set 0 command.  They both visually appear the same.  Turning back "on" with the corresponding ON or set xx command.

Is one better than the other?

**Comment by aderusha** [comment, marked as answer]
Not really, under the hood turning it off does the same thing as setting the dim to 0, so at the end of the day the two are functionally equivalent.
